tst/testinstall/objlist.tst

```diff
@@ -0,0 +1,82 @@
+gap> DeclareCategory("IsTestListObj", IsSmallList);;
```

Comment:
Could you pleas add a brief comment here which explains what is going on below? Something like (assuming I guess right):
> In this test, we set up a custom small list implementation and use it to test that various GAP features work correctly with non-internal lists.